### DISCUSSION TITLE
sending genericTick"106" to reqMktData to get IMPLIEDVOLATILITY but nothing comes back

### DISCUSSION THREAD

**Discussion by ghost** [lbilli/Jib.jl#25] [Q&A] [answered]
trying to get IMPLIEDVOLATILITY  for a symbol  the ib docs [  https://interactivebrokers.github.io/tws-api/tick_types.html](url) tell me it's 

generictick = "106"  and so I enter it in this call

```
Jib.reqMktData(ib, idx, contract, "106", false)
'''

I then look in the callback for tickGeneric but I don't see anything.

  What did I do wrong please?

**Comment by lbilli** [comment, marked as answer]
Hard to say. Make sure the stock does indeed have options.
Here's a typical run:
```julia
julia> stock = Jib.Contract(symbol="AAPL", secType="STK", exchange="SMART", currency="USD")

julia> Jib.reqMktData(ib, 57, stock, "mdoff,104,106", false)

tickReqParams: 57 0.01 9c0001 3
tickGeneric: 57 OPTION_HISTORICAL_VOL 0.36271455051061996
tickGeneric: 57 OPTION_IMPLIED_VOL 0.35363669304385537
marketDataType: 57 REALTIME
tickGeneric: 57 OPTION_IMPLIED_VOL 0.35536941587354764
tickGeneric: 57 OPTION_HISTORICAL_VOL 0.3625721958397805
```

**Reply by ghost** [reply]
thanks now I'm getting IV and HV which is great  but now I'm not getting  LAST, VOLUME. What does "mdoff" do please?  I can't seem to find anything about it in the IB docs.
All the symbols I am looking at have options.

**Reply by lbilli** [reply]
`mdoff` just suppresses the market data ticks BID, ASK, LAST, etc. that are on by default.
If you want those plus HV and IV just use "104,106".

**Reply by ghost** [reply]
Perfect thanks so much.
